samples/contrib/arena-samples/standalonejob/standalone_pipeline.ipynb

```diff
@@ -328,11 +304,15 @@
     "dropout = \"0.8\"\n",
     "model_verison = \"1\"\n",
     "\n",
-    "compiler.Compiler().compile(sample_pipeline,  'standalone.tar.gz')\n",
+    "arguments = {\n",
+    "  'learning_rate': learning_rate,\n",
+    "  'dropout': dropout,\n",
+    "  'model_version': model_version,\n",
+    "}\n",
     "\n",
-    "run = client.run_pipeline(experiment_id, 'mnist', 'standalone.tar.gz', params={'learning_rate':learning_rate,\n",
-    "                                     'dropout':dropout,\n",
-    "                                    'model_version':model_version})\n",
+    "import kfp\n",
+    "client = kfp.Client(host=KUBEFLOW_PIPELINE_LINK)\n",
+    "run = client.create_run_from_pipeline_func(sample_pipeline, arguments=arguments).run_info\n",
```

Comment:
Shall we also specify experiment name here? For it was assigned on the top of the notebook.

Reply:
I think it's not really needed for this notebook. It was required before, but not now. The concept of "experiments" might be an advanced concept that's not normally used.

P.S. This sample is not under sample tests. For notebooks under sample tests, I keep experiment_name for now, but I have a PR which will make it unnecessary (it kind of depends on this PR).